src/components/screens/signMultiSignTransaction/reviewSign/reviewSign.js

```diff
@@ -95,44 +62,16 @@ const ReviewSign = ({
   nextStep,
   history,
   error,
-  dispatch,
   senderAccount,
 }) => {
-  // eslint-disable-next-line max-statements
-  const showSendButton = () => {
+  const isMember = useMemo(() => {
     if (senderAccount.data?.keys) {
-      const isGroupRegistration = transaction.moduleAssetId
```

Comment:
Why were these removed? I added this as a fix in #3579

Reply:
As I observe they were moved to helpers. @reyraa is it right?